### PR TITLE
[Misc] Update version

### PR DESCRIPTION
**Pull request by Accelerator1996**

### CODE REVIEW

File: version.txt [hunk 1]
```diff
@@ -1 +1 @@
-11.0.24.20.8-GA
\ No newline at end of file
+11.0.25.21.9-GA
```
Comment (Accelerator1996) [inline version.txt:1]: diff --git a/version.txt b/version.txt
index 48d075b2d5..b2d0310003 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-11.0.25.21.9-GA
\ No newline at end of file
+11.0.24.20.8-GA
\ No newline at end of file

Comment (Accelerator1996) [inline version.txt:1]:
```
diff --git a/version.txt b/version.txt
index b2d0310003..48d075b2d5 100644
--- a/version.txt
+++ b/version.txt
@@ -1 +1 @@
-11.0.24.20.8-GA
\ No newline at end of file
+11.0.25.21.9-GA
\ No newline at end of file
```

**Review by kuaiwei — APPROVED** [review]
LGTM

**Review by sendaoYan — APPROVED** [review]
/LGTM

**Pull request merged**